debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -268,14 +269,15 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
             coordinator.start(taskContext, this.queue, metadataProvider);
 
             return coordinator;
+        }
```

Comment:
Do not follow this change ? Just seems to have moved and formatting has changed ?

Reply:
inner try doesn't have catch but just finally. The catch block is for the overall try block.